lib/tasks/bugfix.rake

```diff
@@ -107,4 +107,59 @@ namespace :bugfix do
       end
     end
   end
+
+
+  desc "Remove Duplicate Answers"
+  task remove_duplicate_answers: :environment do
+    ## Concat Duplicate Answers
+    ActiveRecord::Base.transaction do
+      plan_ids = ActiveRecord::Base.connection.select_all("SELECT a1.plan_id as plan_id FROM Answers a1 INNER JOIN Answers a2 ON a1.plan_id = a2.plan_id AND a1.question_id = a2.question_id WHERE a1.id > a2.id" ).to_a.map{|h| h["plan_id"]}.uniq
```

Comment:
Could have probably gotten to the list of questions in one query, but if this is working its fine since its really only run once.